CHANGELOG.md

```diff
@@ -1,13 +1,5 @@
 # Changelog
 
-## 09/18/2018 - Version 1.0.0-RC1
```

Comment:
Did this get deleted automatically?

Reply:
I deleted it myself since it's not very accurate, I believe it gets generated automatically. I'll write detailed release notes on 1.0.0 when it's released.